clients/html/src/app/employer-details/employer-details.component.ts

```diff
@@ -180,6 +180,10 @@ export class EmployerDetailsComponent implements OnInit {
       is_late_rate = response['is_late_rate'];
       this.isLateRates = is_late_rate;
       this.isSicCodesEnabled = response['is_sic_codes_enabled'];
+
+      localStorage.setItem('is_benefit_model_enabled', JSON.stringify(response['is_benefit_model_enabled']));
```

Comment:
curious if we can use the response directly, instead of setting in localStorage?

Reply:
since this is another module, we cannot use response directly. Ideal way is to make an api call from the module itself to get this information. but, it seems overkill to make another api call to get this info and also we redirect user to home page on every refresh (so, no problem with bookmarking urls)